packages/components/tests/acceptance/components/hds/application-state-test.js

```diff
@@ -0,0 +1,21 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'dummy/tests/helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+
+module('Acceptance | components/hds/application state', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /components/hds/application-state', async function (assert) {
+    await visit('/components/application-state');
+
+    assert.strictEqual(currentURL(), '/components/application-state');
```

Comment:
[suggestion/$.02/off topic for the actual question we're trying to answer] acceptance tests get expensive in time, so I don't think this test provides a ton of value (we already test that this page renders as we expect in the percy snapshots, so the UR being correct or not isn't super valuable).